src/openrct2/world/Location.hpp

```diff
@@ -13,7 +13,7 @@
 
 #include <algorithm>
 
-#define LOCATION_NULL ((int16_t)(uint16_t)0x8000)
+constexpr const int16_t LOCATION_NULL = -32768;
```

Comment:
Would appreciate a confirmation that this is indeed the same value.

Reply:
Confirmed: http://cpp.sh/6tvae